auslib/web/admin/swagger/api.yaml

```diff
@@ -349,6 +405,160 @@ paths:
               type: string
           examples:
             text/html: "csrf_token: 1491342563##c4e6fef0b978e6c89af9ff1015e67b9ca7c45d14"
+  
+  /rrp/history:
```

Comment:
The endpoints look much improved now, but I'd like to clean up the API a bit further. At the moment, you've got 3 endpoints that all grab history, but the objects in each endpoint are only loosely related. I think there's two ways we can go:
1) Use one endpoint for ALL objects (rules, releases, permissions, scheduled changes, required signoffs -- everything).
2) Use individual endpoints for each type of object (eg: rules would have its own endpoint AND rule scheduled changes would have its own endpoint).

I don't have a strong preference either way, but either would be cleaner than the current endpoints.